Package.swift

```diff
@@ -17,7 +17,9 @@ let package = Package(
             targets: ["MountebankSwift"]
         ),
     ],
-    dependencies: [],
+    dependencies: [
+        .package(url: "https://github.com/apple/swift-docc-plugin", from: "1.3.0")
+    ],
```

Comment:
Is this OK? Just a dep for the project, not for any targets though. I guess this won't be downloaded when you would add MountebankSwift to your project? 

This is as suggested https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/

Reply:
It is only needed if you want to generate the docs outside of Xcode. You can use `Build` > `Build Documentation` within Xcode it self to generate the documentation.